h2o-core/src/main/java/water/util/ArrayUtils.java

```diff
@@ -874,6 +885,12 @@ public static long maxValue(long[] from) {
       if (from[i]>result) result = from[i];
     return result;
   }
+  public static int maxValue(Integer[] from) {
+    int result = from[0];
+    for (int i = 1; i<from.length; ++i)
```

Comment:
in this case both, `++i` or `i++` expressions work the same, however, the convention is `i++`. Personally, I would use java 8 Stream API for finding maximum here, since it's more bulletproof.

Reply:
I switched to stream.  Thanks!